### PR TITLE
Revert "Add community-reviewed label is need to merge"

### LINKED ISSUE #2855: Red check label triage
### Describe what maintenance you would like added.

I'd like to point out that it's a little annoying to see the red check on each and every commit, despite the fact that some of the commits might actually be valid. It makes walking back through looking for valid commits a bit more tedious. Since I like the approach of waiting, is there a way we could still include the labeler workflow but not have this behavior?

_Originally posted by @akaszynski in https://github.com/pyvista/pyvista/issues/2853#issuecomment-1165983311_

### Links to source code.

_No response_

### Pseudocode or Screenshots

```python 
 Insert code here 
 ```
 Or drag your screenshot here! 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by tkoyama010**
Reverts pyvista/pyvista#2851
See https://github.com/pyvista/pyvista/pull/2857

Resolves #2855

### CODE REVIEW

**Review by akaszynski — APPROVED** [review]
Sorry this didn't work out @tkoyama010, but I appreciate your efforts nonetheless.

Let's ask GitHub for a timer for automerge, that would be awesome!

**Comment by adeak** [comment]
Indeed, thanks again @tkoyama010 for labouring with this attempt to make the review workflow smoother.

**Comment by tkoyama010** [comment]
Thanks for the comments. I love challenges like this. The ability to check the required labels in the release notes was a great accomplishment we got this time. I will reflect on this part after merging this.

File: .github/workflows/label.yml [hunk 1]
```diff
@@ -44,18 +43,3 @@ jobs:
       if: startsWith(github.event.pull_request.head.ref, 'release')
       with:
         labels: release
```
Comment (tkoyama010) [inline .github/workflows/label.yml:45]:
```suggestion
        labels: release
- uses: mheap/github-action-required-labels@v2
      with:
        mode: minimum
        count: 1
        labels: "documentation, docker, maintenance, ignore-for-release, enhancement, bug, release"
```

File: .github/workflows/label.yml [hunk 2]
```diff
@@ -44,18 +44,8 @@ jobs:
...
-    - uses: mheap/github-action-required-labels@v2
+- uses: mheap/github-action-required-labels@v2
```
Comment (tkoyama010) [inline .github/workflows/label.yml:47]:
```suggestion
    - uses: mheap/github-action-required-labels@v2
```

**Comment by tkoyama010** [comment]
For Reverts, no review is required, so the 24-hour rule is not applied and the merge is performed.

**Pull request merged**